src/test/java/seedu/address/logic/commands/GemCommandTest.java

```diff
@@ -23,18 +24,75 @@
 
 public class GemCommandTest {
 
-    public static final Path COMPILED_PATH_1 = Paths.get("src/test/data/GemCommandTest/compiledsem1GEmodules.json");
-    public static final Path COMPILED_PATH_2 = Paths.get("src/test/data/GemCommandTest/compiledsem2GEmodules.json");
+    public static final Path GEH_PATH_1 = Paths.get("src/test/data/GemCommandTest"
+        + "/sem1GEHmodules.json");
+    public static final Path GET_PATH_1 = Paths.get("src/test/data/GemCommandTest"
+        + "/sem1GETmodules.json");
+    public static final Path GES_PATH_1 = Paths.get("src/test/data/GemCommandTest"
+        + "/sem1GESmodules.json");
+    public static final Path GEQ_PATH_1 = Paths.get("src/test/data/GemCommandTest"
+        + "/sem1GEQmodules.json");
+    public static final Path GER_PATH_1 = Paths.get("src/test/data/GemCommandTest"
+        + "/sem1GERmodules.json");
+
+    public static final Path GEH_PATH_2 = Paths.get("src/test/data/GemCommandTest"
+        + "/sem2GEHmodules.json");
+    public static final Path GET_PATH_2 = Paths.get("src/test/data/GemCommandTest"
+        + "/sem2GETmodules.json");
+    public static final Path GES_PATH_2 = Paths.get("src/test/data/GemCommandTest"
+        + "/sem2GESmodules.json");
+    public static final Path GEQ_PATH_2 = Paths.get("src/test/data/GemCommandTest"
+        + "/sem2GEQmodules.json");
+    public static final Path GER_PATH_2 = Paths.get("src/test/data/GemCommandTest"
+        + "/sem2GERmodules.json");
+
     public static final Path TEST_GEH_SEM1_PATH = Paths.get("src/main/resources/data/GEM/GEHs1.json");
     public static final Path TEST_GEH_SEM2_PATH = Paths.get("src/main/resources/data/GEM/GEHsem2.json");
     private Model model;
     private GemCommand gemCommand = new GemCommand();
     private ObservableList<Module> testModules;
+    private ObservableList<Module> gehTestModules;
+    private ObservableList<Module> getTestModules;
+    private ObservableList<Module> gesTestModules;
+    private ObservableList<Module> geqTestModules;
+    private ObservableList<Module> gerTestModules;
+
     public void setUpTestModules(Path path) throws IOException, DataConversionException {
         JsonGradPadStorage storage = new JsonGradPadStorage(path);
         ReadOnlyGradPad gradPad = storage.readGradPad().get();
         testModules = gradPad.getModuleList();
     }
+
+    public void setUpGehTestModules(Path path) throws IOException, DataConversionException {
+        JsonGradPadStorage storage = new JsonGradPadStorage(path);
+        ReadOnlyGradPad gradPad = storage.readGradPad().get();
+        gehTestModules = gradPad.getModuleList();
+    }
```

Comment:
I think we can reuse the testModules method and attribute so we can reduce the clutter and make the code cleaner? HAHA since the setUp methods all have roughly the same function